modules/core/src/main/scala/dev/profunktor/redis4cats/effect/FutureLift.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018-2021 ProfunKtor
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.profunktor.redis4cats.effect
+
+import cats.ApplicativeThrow
+import cats.effect.Async
+import cats.syntax.all._
+import io.lettuce.core.{ ConnectionFuture, RedisFuture }
+
+import java.util.concurrent._
+
+private[redis4cats] trait FutureLift[F[_]] {
```

Comment:
Is this supposed to be package-private? It seems to be part of the public API of RedisClient.

Reply:
Yeah it should be private. I removed it from the `RedisClient.apply[F]`, is it somewhere else publicly available?

Reply:
It's also publicly available as a field in `MkRedis`.

Reply:
I'll make them private 👍🏾

Reply:
I guess `MkRedis` may as well be sealed in that case, since users can't implement it.